### PR TITLE
Upgrade to latest dependencies

### PR DESCRIPTION
**Pull request by dzfranklin**
Because cargo doesn't support multiple versions of native dependencies
users of cairo and poppler must use versions that depend on the same
version of glib. I had to bump all the dependencies in unison to get them to work together.

Before this change a user would have to use cairo 0.6 if they needed
poppler, now they can upgrade to cairo 0.8.1.

Cairo removed glib::error::Error::wrap in the commit ["Fix some clippy warnings"], and I didn't see any recommended replacement. Browsing through their code they seemed to use the unsafe fn `glib::translate::from_glib_full` for the same purpose. I am unsure if that is actually the right way to convert such an error.

["Fix some clippy warnings"]: https://github.com/gtk-rs/glib/commit/c80276b3d72a9ef679b2fef1b9fce3b1ab55e19e#diff-9a269209bed069ffb36d5565a91c9840L89

### CODE REVIEW

**Comment by denysvitali** [comment]
LGTM, but I'd rather not have unsafe functions here. Do you know of any alternative?

I might look closely into this in the following days if you do not have any priority in getting it merged. Contributions / Ideas from others are always welcome

**Comment by dzfranklin** [comment]
This isn't a priority for me, I'm happy to wait.

I have essentially zero experience with glib or glib-rs. Just because I didn't see a safe way to do it doesn't at all mean there isn't one.

**Comment by RealPacket** [comment]
updated in commit 964fd4b0ada2d2801395b54e235a6dd0f89b9213

**Comment by denysvitali** [comment]
Can you please rebase your branch?
I'll then run the CI and if the tests pass we can bump the version and merge it :)

File: Cargo.toml [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-cairo-rs = { version= "0.6.0", features = ["png", "pdf"]}
-cairo-sys-rs = "0.8.0"
-glib = "0.7.1"
-glib-sys = "0.8.0"
-gobject-sys = "0.8.0"
+poppler-sys = { version = "0.1.0", path = "poppler-sys" }
+cairo-rs = { version = "0.9.1", features = ["png", "pdf"] }
+glib = "0.10.1"
+glib-sys = "0.10.0"
+gobject-sys = "0.10.0"
```
Comment (RealPacket) [inline Cargo.toml:18]: even more dependencies...

**Comment by RealPacket** [comment]
The dependency tree will probably explode if you used this...

**Pull request closed without merging**